site-setup/learn_article.ftl

```diff
@@ -94,32 +93,16 @@ html{box-sizing:border-box;-ms-overflow-style:scrollbar}*,::after,::before{box-s
 				</div>
 
 				<div class="doc-nav">
-					{% include "highlighting-alert.html" %}
-
-					{% if parents|length > 0 %}
```

Comment:
We'll need to replicate this logic instead of just removing it (now it always just shows "Go Back" and the url is empty) - can you think through what new fields we'll need in the article structure?